src/serializationHelpers.ts

```diff
@@ -29,56 +48,187 @@ export function serializeList<T>(
     return Buffer.concat(buffers);
 }
 
+/**
+ * Encodes a 128 bit signed integer to a Buffer using big endian.
+ * @param value a 128 bit integer
+ * @useLittleEndian a boolean value false to use big endian else little endian.
```

Comment:
```suggestion
 * @param useLittleEndian a boolean value, if not given, the value is serialized in big endian.
```

Reply:
Also the return description says the serialization is big endian, but that should be changed.

Reply:
This applies for most of these encode functions